main.py

```diff
@@ -1,7 +1,49 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    # Create genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
Typographical Error: The genre name 'Dramma' should be corrected to 'Drama' initially to prevent the need for an update later.